### PR TITLE
better sidebar animation

### PR DESCRIPTION
**Pull request by kylesferrazza**

### CODE REVIEW

File: app/javascript/components/navbar/ExamNavbar.tsx [hunk 1]
```diff
@@ -20,41 +22,58 @@ import Scratch from '@hourglass/containers/navbar/Scratch';
...
+    expanded,
     Icon,
     label,
     eventKey,
     children,
     className = 'bg-secondary text-light',
     direction = 'down',
   } = props;
-  const iconSize = '1.5em';
+  // const dExpanded = displayClass(expanded);
```
Comment (kylesferrazza) [inline app/javascript/components/navbar/ExamNavbar.tsx:48]: We can remove this now

File: app/javascript/components/navbar/ExamNavbar.css [hunk 2]
```diff
@@ -10,13 +10,56 @@
...
+.sidebar-expanded {
+  min-width: 500px;
+}
+
+.width-0 {
+  width: 0;
+}
+
+.collapse:not(.width):not(.show) {
+  display: none; }
```
Comment (kylesferrazza) [inline app/javascript/components/navbar/ExamNavbar.css:30]: awk. brace here

File: app/javascript/components/navbar/ExamNavbar.tsx [hunk 3]
```diff
@@ -20,41 +22,58 @@ import Scratch from '@hourglass/containers/navbar/Scratch';
...
+  onSectionClick: (eventKey: string) => void;
+  expanded: boolean;
   Icon: IconType;
   label: string;
   className?: string;
   eventKey: string;
   direction?: 'up' | 'down';
 }
 
+const iconSize = '1.5em';
```
Comment (kylesferrazza) [inline app/javascript/components/navbar/ExamNavbar.tsx:35]: Should we make `1.5em` a project-wide constant for icon sizes?

File: app/javascript/components/navbar/ExamNavbar.tsx (hunk 1 above)
Comment (blerner) [reply app/javascript/components/navbar/ExamNavbar.tsx:48]: Yeah, I thought I'd done so already.  I must have only removed one of them, but this one was commented out and didn't get noticed.

File: app/javascript/components/navbar/ExamNavbar.css (hunk 2 above)
Comment (blerner) [reply app/javascript/components/navbar/ExamNavbar.css:30]: Yup.

File: app/javascript/components/navbar/ExamNavbar.tsx (hunk 3 above)
Comment (blerner) [reply app/javascript/components/navbar/ExamNavbar.tsx:35]: Probably, yes.  Where ought that go?

Comment (kylesferrazza) [reply app/javascript/components/navbar/ExamNavbar.tsx:35]: `helpers.ts` is kind of a catchall right now and is probably going to be split up later..

Maybe we want a `@hourglass/constants` with `export const ICON_SIZE = '1.5em';`

Comment (blerner) [reply app/javascript/components/navbar/ExamNavbar.tsx:35]: That sounds reasonable

Comment (kylesferrazza) [reply app/javascript/components/navbar/ExamNavbar.tsx:35]: I think the linter will complain about wanting you to use the default export - just add a `// eslint-disable-next-line import/prefer-default-export` before the `ICON_SIZE` line until we have more constants.

**Pull request merged**